examples/example_gtk3_opengl3/main.cpp

```diff
@@ -0,0 +1,141 @@
+// ImGui - standalone example application for GLFW + OpenGL 3, using programmable pipeline
+// If you are new to ImGui, see examples/README.txt and documentation at the top of imgui.cpp.
+// (GLFW is a cross-platform general purpose library for handling windows, inputs, OpenGL/Vulkan graphics context creation, etc.)
+// (GL3W is a helper library to access OpenGL functions since there is no standard header to access modern OpenGL functions easily. Alternatives are GLEW, Glad, etc.)
+
+#include "imgui.h"
+#include "imgui_impl_gtk3.h"
+#include "imgui_impl_opengl3.h"
+#include <stdio.h>
```

Comment:
Why `stdio.h` here? I think you didn't use this at all.

Reply:
Probably some leftover debugging :)

Reply:
Actually it's just copied over from the other main.cpp.

Reply:
:) It actually looks good to me. Let's see what @ocornut thinks about this.